(dp1
S'output'
p2
S"<type 'exceptions.IndexError'> list index out of range"
p3
sS'layer'
p4
S'C:\\Python27\\Lib\\site-packages\\web2py\\applications\\viewer\\controllers/show.py'
p5
sS'code'
p6
S"# -*- coding: utf-8 -*-\n\ndef index():\n    redirect(URL('table'))\n\ndef table():\n    '''\n    auth_prefix = 'auth_'    # hide authorizations tables\n    len_prefix = len(auth_prefix)\n    table = request.args\n    if table not in filter(lambda candidate: candidate[:len_prefix]!=auth_prefix, db.tables):\n        for table in db.tables:\n            if table[:len_prefix]!=auth_prefix:\n                break\n    '''\n    table = request.args and request.args[0] or dbv.tables[0]\n    import pdb;pdb.set_trace()\n    grid = SQLFORM.smartgrid(\n            dbv[table]\n            )\n\n    return dict(grid=grid)\n\nresponse._vars=response._caller(table)\n"
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "C:\\Python27\\Lib\\site-packages\\web2py\\gluon\\restricted.py", line 224, in restricted\n    exec ccode in environment\n  File "C:\\Python27\\Lib\\site-packages\\web2py\\applications\\viewer\\controllers/show.py", line 24, in <module>\n  File "C:\\Python27\\Lib\\site-packages\\web2py\\gluon\\globals.py", line 393, in <lambda>\n    self._caller = lambda f: f()\n  File "C:\\Python27\\Lib\\site-packages\\web2py\\applications\\viewer\\controllers/show.py", line 16, in table\n    table = request.args and request.args[0] or dbv.tables[0]\nIndexError: list index out of range\n'
p11
s.